web-app/src/app/screens/FeedSubmission/index.tsx

```diff
@@ -111,7 +111,8 @@ function Component(): React.ReactElement {
 
 export default function Home(): React.ReactElement {
   const { config } = useRemoteConfig();
-  if (config.enableFeedSubmissionStepper) {
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+  if (config.enableFeedSubmissionStepper || !!window.Cypress) {
```

Comment:
I would avoid references to the Cypress window in production code as much as possible.